cfpb/consumerfinance.gov
# Provide translation of skip link

## Changes

- Provide Spanish language translation of skip link "Skip to main content"


## How to test this PR

1. Turn on VoiceOver and visit /es/ or another Spanish language page and tab into the content to show the skip link (easiest may be to click in the header and Shift + tab to reverse back into the skip link). It should be read using the Spanish language voice.


## Screenshots

<img width="437" alt="Screen Shot 2021-07-29 at 5 33 22 PM" src="https://user-images.githubusercontent.com/704760/127568782-23c253b5-b172-4eaa-8981-9a4b09bc1347.png">
